src/lib/Libattr/master_resv_attr_def.xml

```diff
@@ -869,6 +869,23 @@
          <ECL>NULL_VERIFY_VALUE_FUNC</ECL>
       </member_verify_function>
    </attributes>
+   <attributes>
+      <member_index>RESV_ATR_duration_revert</member_index>
+      <member_name>ATTR_resv_duration_revert</member_name>
+      <member_at_decode>decode_time</member_at_decode>
+      <member_at_encode>encode_l</member_at_encode>
+      <member_at_set>set_l</member_at_set>
+      <member_at_comp>comp_l</member_at_comp>
+      <member_at_free>free_null</member_at_free>
+      <member_at_action>NULL_FUNC</member_at_action>
+      <member_at_flags>ATR_DFLAG_MGRD</member_at_flags>
+      <member_at_type>ATR_TYPE_LONG</member_at_type>
+      <member_at_parent>PARENT_TYPE_RESV</member_at_parent>
+      <member_verify_function>
+         <ECL>verify_datatype_time</ECL>
```

Comment:
No need for this, this value is set by the server.

Reply:
I'll make both server only attributes